services/core/MongodbTaggingService/mongodb/resources/categories.csv

```diff
@@ -0,0 +1,27 @@
+name,description
```

Comment:
Would this and the other "resources" be better placed at a root level of the repository rather than packaged with the agents?

Reply:
I put it in two location since in theory they could be customized - adding new tags, categories etc. Also so that I could use resource_string method from package_resources. what do you think?

Reply:
I think that if the user wants to customize the csv then they should have a place to pass in the categories during runtime...i.e I don't however think that this is very likely to happen.  As with the resource_string option I think the volttron_root path would be more maintainable.